assert/assertions_test.go

```diff
@@ -1499,6 +1499,15 @@ Diff:
 	Equal(t, expected, actual)
 }
 
+func TestTimeEquality(t *testing.T) {
```

Comment:
Would it make sense to have a more explicit test name which very apparently describes what we're testing? A suggestion would be: `TestTimeEqualityFormatting`.

Reply:
Updated.